.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: true
+contact_links:
+  - name: 🤝 Community Support
+    url: https://github.com/YCloudYUSA/yusaopeny-project/discussions
+    about: Ask questions and discuss with the community
+  - name: 📖 Documentation
+    url: https://github.com/YCloudYUSA/yusaopeny/tree/main/docs
+    about: Browse the distribution documentation
```

Comment:
We use `git@github.com:YCloudYUSA/yusaopeny_docs.git` for docs -> https://ds-docs.y.org/ - needs rethink and update